### ISSUE TITLE
Option to drop database before import

### ISSUE THREAD

**Issue by synth** [sgruhier/capistrano-db-tasks#117]
The import cmd (at least for mysql) is 

`mysql #{credentials} -D #{database} < #{file}`

which will result in the file being appended to the current database. 

Is there an option to drop the database before import? If not, can there be?

**Comment by numbata** [comment]
Unfortunately, there isn't an option to drop database for mysql now.

But I think I can do it. "pgsql import" command already try to drop database before import.

Also I saw case where mysql user hasn't "create database" permissions. And we should be careful in "drop database" implementation.

**Comment by kakra** [comment]
I don't think it should ever drop the database - it's too dangerous. The DB may have unrelated table data outside of the Rails model scope. The pros and cons of such a setup may be discussed elsewhere but for historic reasons I am running such a database with one project. You can instruct `mysqldump` to add `DROP TABLE IF EXISTS` during export instead. It should be much more safe. Also, it may be better to wrap the export into one transaction, so on import it is either imported completely or aborted completely.
